tests/foreman/cli/test_discoveryrule.py

```diff
@@ -240,75 +283,37 @@ def test_positive_create_disabled_rule(self, discoveryrule_factory):
         rule = discoveryrule_factory(options={'enabled': 'false'})
         assert rule.enabled == 'false'
 
-    @pytest.mark.tier3
+    @pytest.mark.tier1
+    @pytest.mark.parametrize('host_name', **parametrized(invalid_hostnames_list()))
     @pytest.mark.parametrize('name', **parametrized(invalid_values_list()))
-    def test_negative_create_with_invalid_name(self, name, discoveryrule_factory):
-        """Create Discovery Rule with invalid names
+    def test_negative_create_rule_with_discovery_params(
+        self, name, host_name, discoveryrule_factory
+    ):
+        """Create Discovery Rule with negative parameters.
 
-        :id: a0350dc9-8f5b-4673-be88-a5e35d1f8ca7
+        :id: 0906cf64-ed0b-49af-844f-1af22f81ab94
 
         :expectedresults: Error should be raised and rule should not be created
 
         :CaseImportance: Medium
-
-        :CaseLevel: Component
-
-        :parametrized: yes
         """
         with pytest.raises(CLIFactoryError):
             discoveryrule_factory(options={'name': name})
 
-    @pytest.mark.tier3
-    @pytest.mark.parametrize('name', **parametrized(invalid_hostnames_list()))
-    def test_negative_create_with_invalid_hostname(self, name, discoveryrule_factory):
-        """Create Discovery Rule with invalid hostname
-
-        :id: 0ae51085-30d0-44f9-9e49-abe928a8a4b7
-
-        :expectedresults: Error should be raised and rule should not be created
-
-        :CaseImportance: Medium
-
-        :CaseLevel: Component
-
-        :BZ: 1378427
-
-        :parametrized: yes
-        """
         with pytest.raises(CLIFactoryError):
-            discoveryrule_factory(options={'hostname': name})
-
-    @pytest.mark.tier3
-    def test_negative_create_with_too_long_limit(self, discoveryrule_factory):
-        """Create Discovery Rule with too long host limit value
-
-        :id: 12dbb023-c963-4ead-a81e-ad53033de947
+            discoveryrule_factory(options={'hostname': host_name})
 
-        :expectedresults: Validation error should be raised and rule should not
-            be created
-
-        :CaseImportance: Medium
-        """
         with pytest.raises(CLIFactoryError):
             discoveryrule_factory(options={'hosts-limit': '9999999999'})
 
-    @pytest.mark.tier1
-    def test_negative_create_with_same_name(self, discoveryrule_factory):
-        """Create Discovery Rule with name that already exists
-
```

Comment:
this test wasn't parametrized earlier, I don't think this needs to tested for each value from  invalid_values_list

Reply:
same as above